lib/matplotlib/font_manager.py

```diff
@@ -929,6 +940,43 @@ def set_fontconfig_pattern(self, pattern):
             else:
                 getattr(self, "set_" + key)(val)
 
+    def get_math_fontfamily(self):
+        """
+        Return the name of the font family used for math text.
+
+        The default font is :rc:`mathtext.fontset`.
+        """
+        if self._math_fontfamily is None:
+            return rcParams['mathtext.fontset']
+        return self._math_fontfamily
+
+    def set_math_fontfamily(self, math_fontfamily):
```

Comment:
This and `Text.set_math_fontfamily()` should use consistent parameter names.

I slightly prefer
```suggestion
    def set_math_fontfamily(self, fontfamily):
```
because the other one is very verbose.